hack/generate-metallb-manifests.sh

```diff
@@ -63,11 +63,36 @@ yq e --inplace '. | (select(.kind == "DaemonSet" and .metadata.name == "speaker"
 yq e --inplace '. | (select(.kind == "DaemonSet" and .metadata.name == "speaker") | .spec.template.spec.initContainers[] | select(.name == "cp-metrics").image)|="{{.SpeakerImage}}"' ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
 yq e --inplace '. | select(.kind == "Deployment" and .metadata.name == "controller" and .spec.template.spec.containers[0].name == "controller" and .spec.template.spec.securityContext.runAsUser == "65534").spec.template.spec.securityContext|="'"$(< ${METALLB_SC_FILE})"'"' ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
 yq e --inplace '. | select(.metadata.namespace == "metallb-system").metadata.namespace|="{{.NameSpace}}"' ${FRR_MANIFESTS_DIR}/${FRR_MANIFESTS_FILE}
+
+# kube-rbac-proxy modifications
```

Comment:
This is becoming un-maintainable. Not sure it's worth the effort anymore vs maintaining the manifests manually.
It's really hard to follow what part is doing what in order to get the final result of manifests under bindata.

I'm in favor of removing everything and maintaining manually. Now we have e2e coverage that should warn us in case we missed something.
We can defer this in another PR.